src/inverseRobot.js

```diff
@@ -24,7 +24,21 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const arrOfEntries = Object.entries(robot);
+  const correctRobot = {};
+
+  for (const entry of arrOfEntries) {
+    const key = entry[1];
+    const value = entry[0];
```

Comment:
You may try destructuring here

```suggestion
  for (const [key, value] of arrOfEntries) {
```